onnxscript/evaluator.py

```diff
@@ -7,19 +7,17 @@
 import abc
 import contextlib
 import pprint
+import typing
 from typing import Any, Optional
 
 import numpy as np
 import onnx
-import onnxruntime as ort
-from onnxruntime.capi.onnxruntime_pybind11_state import (
-    Fail,
-    InvalidArgument,
-    InvalidGraph,
-)
 
 from onnxscript import autocast, irbuilder, onnx_opset, tensor, utils, values
 
+if typing.TYPE_CHECKING:
+    import onnxruntime as ort
```

Comment:
Just wondering: why is this necessary?

Reply:
This is for type annotation (for `_cache_models`). ort will be imported by type checkers but not in run time, so we don't fail if the environment doesn't have ort.